tests/cpp_methods/unittest_cpp_methods.cc

```diff
@@ -15,6 +20,9 @@
 #include <nnstreamer-capi-private.h>
 
 #include <tensor_filter_cpp.h>
+#include "../unittest_util.h"
```

Comment:
Q: dependency of unittest_util includes the test directory.
```
  unittest_util_dep = declare_dependency(link_with: unittest_util_shared,
    dependencies: nnstreamer_base_deps,
    include_directories: include_directories('.')
  )
```

Reply:
Fixed.